app/src/main/java/com/spencerpages/collections/LargeCents.java

```diff
@@ -0,0 +1,120 @@
+package com.spencerpages.collections;
+
+import android.database.sqlite.SQLiteDatabase;
+
+import com.coincollection.CoinPageCreator;
+import com.coincollection.CoinSlot;
+import com.coincollection.CollectionInfo;
+import com.coincollection.CollectionListInfo;
+import com.spencerpages.R;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+public class LargeCents extends CollectionInfo {
+
+    public static final String COLLECTION_TYPE = "LargeCents";
+
+    private static final Object[][] COIN_IDENTIFIERS = {
+            {"Flowing Hair Chain Reverse", R.drawable.annc_us_1793_1c_flowing_hair_cent},
+            {"Flowing Hair Wreath Reverse", R.drawable.annc_us_1793_1c_flowing_hair_cent},
```

Comment:
Should "Reverse" be included? Seems like it should be dropped
```suggestion
            {"Flowing Hair Chain", R.drawable.annc_us_1793_1c_flowing_hair_cent},
            {"Flowing Hair Wreath", R.drawable.annc_us_1793_1c_flowing_hair_cent},
```